server/config.ts

```diff
@@ -154,5 +154,8 @@ export default {
       agent: new AgentConfig(Number(get('FRONTEND_COMPONENTS_TIMEOUT_RESPONSE', 3000))),
     },
   },
+  features: {
+    combinedSearch: get('FEATURE_COMBINED_SEARCH', 'false', requiredInProduction) === 'true',
```

Comment:
Because you've set it to be `requiredInProduction` you'll have to add it to values files (e.g. for dev & staging) otherwise I _think_ it'll fail to deploy

Reply:
Why is there no `this` emoji on GH 😁